aip-console-jenkins/src/main/resources/io/jenkins/plugins/aipconsole/AddVersionBuilder/config.properties

```diff
@@ -30,3 +30,7 @@ domainName=Domain Name
 domainName.descr=Name of the domain to assign to this application. Will be created if it doesn't exists in AIP Console. If no specified, no domain will be assigned or created.
 processImaging=Publish to Imaging
 processImaging.descr=Publish analysis data to Imaging if an Imaging instance is defined in AIP Console
+aipConsoleUrl=AIP Console URL
+aipConsoleUrl.descr=The root URL to access AIP Console
```

Comment:
i propose to change to "AIP Console URL if different than the global configuration"